backend/src/main/java/org/eclipse/tractusx/semantics/registry/service/ShellService.java

```diff
@@ -296,6 +297,33 @@ public GetAllAssetAdministrationShellIdsByAssetLink200Response findExternalShell
       }
    }
 
+   @Transactional( readOnly = true )
+   public SearchAllShellsByAssetLink200Response findExternalShellIdsByAssetLinkByExactMatch( Set<ShellIdentifier> shellIdentifiers,
```

Comment:
Is the class SearchAllShellsByAssetLink200Response defined in the official openAPI file? 
Because the logic etc. is the same like in findExternalShellIdsByIdentifiersByExactMatch(). 
Only the same result inserted in different class which has the same attributes.

Reply:
@tunacicek I checked it, but since it is autogenerated, so I think it has to be like that.